modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/TeleportSign.java

```diff
@@ -29,10 +31,18 @@ public final void onSignClick(PlayerInteractEvent event) {
         if (CraftManager.getInstance().getCraftByPlayer(event.getPlayer()) == null) {
             return;
         }
-        String[] numbers = ChatColor.stripColor(sign.getLine(1)).split(",");
-        int tX = Integer.parseInt(numbers[0]);
-        int tY = Integer.parseInt(numbers[1]);
-        int tZ = Integer.parseInt(numbers[2]);
+        
+        int tX = sign.getX(); int tY = sign.getY(); int tZ = sign.getZ();
+        try {
+	        String[] numbers = ChatColor.stripColor(sign.getLine(1)).replaceAll(" ", "").split(",");
+	        tX = Integer.parseInt(numbers[0]);
+	        tY = Integer.parseInt(numbers[1]);
+	        tZ = Integer.parseInt(numbers[2]);
+        } catch (IndexOutOfBoundsException | NumberFormatException e) {}
```

Comment:
Don't ignore the number format exception, and check the bounds on the array rather than catching and IndexOutOfBoundsException`. Exceptions are expensive to handle.